source/administration/config-settings.rst

```diff
@@ -1107,10 +1109,10 @@ Enable Encryption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 **True**: Mattermost will decrypt SAML Assertions encrypted with your Service Provider Public Certificate.
 
-**False**: Encyption is disabled.
+**False**: Encyption is disabled. Not recommended for production environments. For testing only.
 
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"Encrypt": false`` with options ``true`` and ``false`` for above settings respectively.                                  |
+| This feature's ``config.json`` setting is ``"Encrypt": true`` with options ``true`` and ``false`` for above settings respectively.                                   |
```

Comment:
~for above settings respectively.~
Removing those words would make it inconsistent with the rest of the doc, but eventually I think they should all be removed as they are superfluous and add very little or nothing to understanding the doc.